.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,241 @@
+name: build-and-publish
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+env:
+  snap7-archive-url: https://sourceforge.net/projects/snap7/files/1.4.2/snap7-full-1.4.2.7z/download
+jobs:
+  linux-build:
+    name: Build binary wheels for linux
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - arch: x86_64
+            makefile: x86_64_linux.mk
+          - arch: i686
+            makefile: i386_linux.mk
```

Comment:
who still uses i386?!? :)

Reply:
This is for the 32bit version of the library. If we don't need it, it will make thing easier. Do we need 32-bit versions?

Reply:
if it is trivial to keep support then we should keep it.